cves/kernel/CVE-2017-18549.yml

```diff
@@ -241,6 +265,13 @@ interesting_commits:
     note:
   - commit:
     note:
+  - commit: 5cc973f09e21b5a2f746307641879bc9f1da623b
+    note: |
+      This was this commit right before the error was fixed. This also involves a memory leak into the userspace. 
+      Here, a variable was uninitialized and its contents were being written to the userspace. Based on the nature of the error, 
+      it seems that it is quite common. The issue is that an uninitialized variable will always contain a garbage value from the kernel stack. 
+      This 'garbage' can contain important information which could be used maliciously if it is passed to the userspace, 
```

Comment:
I would put a copy of this last sentence into your description.

Reply:
statement added to description